files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -153,6 +153,18 @@
 #
 - command: systemctl mask systemd-journald-audit.socket
 
+#
+# Create directory to store logs of all terminal activity by delphix
+# super users. These logs can be retrieved via a customer-facing API
+# for auditing purposes.
+#
+- file:
+    path: /var/delphix/log/session-logs
+    state: directory
+    group: staff
+    mode: 0775
```

Comment:
Can you clarify why these specific permissions? This directory will hold an audit trail of all shell activity performed by the delphix user. Ignoring for a moment that the delphix user is the only user that is allowed to login interactively in production systems, this information should not be readily available to anyone other than the delphix user, and root (as well as any APIs you design to expose this on purpose). As such, it feels like this should be `0700`?

Reply:
I wasn't sure exactly how locked down these files should be, but given what you describe, yes `0700` sounds better.